README.md

```diff
@@ -292,31 +292,33 @@ If you have received the package and only want to build and run the docker conta
     ```
 
 ### Running the Signing Service
+Set `ETHEREUM_RPIVATE_KEY`, `SIGNING_SERVICE_PRIVATE_KEY` and if you want `SENTRY_DSN` to bash variables encoded in base64 before proceeding.
 To run it in a docker container with access to your local network interface, run:
 
 ```bash
 docker run                                                                               \
-    --rm                                                                                 \
+    --env         ETHEREUM_PRIVATE_KEY                                                   \
+    --env         SIGNING_SERVICE_PRIVATE_KEY                                            \
+    --env         SENTRY_DSN                                                             \
     --network     host                                                                   \
     --hostname    signing-service                                                        \
     --name        signing-service                                                        \
     --restart     on-failure                                                             \
     concent-signing-service                                                              \
-        concent.golem.network                              #host                         \
-        85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2Us   #concent_public_key in base64 \
-        --concent-cluster-port                  9055                                     \             \
+        concent.golem.network                                                            \
+        85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2Us                                 \
+        --concent-cluster-port                  9055                                     \
         --ethereum-private-key-from-env                                                  \
         --signing-service-private-key-from-env                                           \
         --sentry-dsn-from-env
 ```
 
 This assumes that:
 - The service can connect to a Concent cluster at `concent.golem.network:9055`.
-- There's a file called `contract.key` in your local directory and that it contains the private key that can sign Ethereum transactions from the account owned by Concent contract.
+- `85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2Us` is Concent's public key encoded in base64.
+- The `--ethereum-private-key-from-env` and `--signing-service-private-key-from-env` are required.
```

Comment:
> The `--ethereum-private-key-from-env` and `--signing-service-private-key-from-env` are required.

This not necessary. We're only showing an example here. The person setting it up can always see the full set of options using `--help`.

Reply:
Done.